internal/graph/graph.go

```diff
@@ -531,7 +532,8 @@ func joinLabels(s *profile.Sample) string {
 		}
 	}
 	sort.Strings(labels)
-	return strings.Join(labels, `\n`)
+	// join labels with a newline: this can be a bit confusing for labels with newlines
```

Comment:
nit: please make comments full sentences (titlecase, trailing dot).

Also here and in other places please make comments fit 80 columns, like the rest of the pprof codebase.

Reply:
Okay! I've edited this to describe the escaping change you mentioned below.